testing/wireguard-tools/APKBUILD

```diff
@@ -0,0 +1,65 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+
+pkgname=wireguard-tools
+pkgver=0.0.20170223
+pkgrel=0
+pkgdesc="Next generation secure network tunnel: userspace tools"
+arch='all'
+url='https://www.wireguard.io'
+license="GPLv2"
+makedepends="libmnl-dev sparse"
```

Comment:
it was never a hard dependency to the userspace tools (only a build time depends)

removed in https://github.com/alpinelinux/aports/pull/954